### ISSUE TITLE
Select multiple edges by edge value

### ISSUE THREAD

**Issue by jpbell27** [datastorm-open/visNetwork#240]
Hi- sorry if this has been answered before, but I can't find it listed in previous issues.

Is there a way to select edges based on an edge value (like nodes can be selected by ID or other defined options)? 

I'm attempting to figure out a way to display/highlight groups of edges based on a pre-defined edge value, but cannot seem to find a way to make this work in the visNetwork package. Any suggestions or help would be greatly appreciated.

**Comment by kgellatl** [comment]
I also agree with this feature! In my context we have edges that change across conditions. It would be very helpful to only selected edges (e.g. based on the state of their change UP / DOWN in my case)
